### ISSUE TITLE
Picking a framework

### ISSUE THREAD

**Issue by gandie** [starvingcats/audiofizzle#2]
To create a mobile app we will a need a suitable framework. Because we want to use web technology to make cross-plattforming more painless, a framework like Apache Cordova comes to my mind. Are there more frameworks like this?

What's important?
- The created app must use web technology: HTML, JS, CSS
- Projects created within the framework must be buildable from at least linux and windows systems
- The framework must handle all tasks neccessary to build the mobile app - at least Android must be available as target OS

**Long story short:** Are there any suitable Cordova alternatives?

**Comment by gandie** [comment]
Cordova environment installation woks fine on Ubuntu18 compared to Ubuntu16.
Lets use cordova.
